docs/README.md

```diff
@@ -320,6 +322,45 @@ Notes:
 * All regular expressions are case-sensitive by default. If you don't want to differentiate between upper and lower-case letters, use non-capturing groups with a flag: `(?flags:exp)`, for example: `(?i:my_case_insensitive_regex)`
 * LogDNA also provides post-ingestion <a href="https://docs.logdna.com/docs/excluding-log-lines" target="_blank">exclusion rules</a> to control what log data is displayed and stored in LogDNA.
 
+### Configuration for Kubernetes Metadata Filtering
+
+In addition to being able to write custom regex expressions, the Agent can be configured to filter log lines associated with various Kubernetes resources. For example, include only log lines coming from a given namespace. This can be useful because writing a custom regex expression for filtering out logs coming from a specific Kubernetes resources requires the user to know how Kubernetes formats it's log files. Using the Kubernetes filtering configuration makes this much easier.
+
+You can configure the Kubernetes metadata filtering via the following environment variables:
+
+* include *only* log lines associated with a specific resource with `LOGDNA_K8S_METADATA_INCLUSION`
+* exclude log lines associated with specific resource with `LOGDNA_K8S_METADATA_EXCLUSION`
+
+Currently, we support filtering on four different fields: namespace, pod, label and annotation. The following shows the nomenclature for each of the supported fields:
+
+```
+  namespace:<value>
+  pod:<value>
```

Comment:
@c-nixon let me know your thought on using `pod` vs `name` based on our discussion around the metadata pod object.

Reply:
I still have a slight preference for `name` since that's what it's called in the k8s manifest